.gitignore

```diff
@@ -226,3 +226,6 @@ CVS
 .ssh/*
 id_rsa*
 id_dsa*
+
+# Carthage
+Carthage/Build
```

Comment:
I see what you mean, but I prefer to be consistent with the more popular libs and the widely used github/gitignore swift file. I don't think the trailing slash is useful in this case because Carthage/Build will be a directory for the foreseeable future.